Code/RDGeneral/Dict.h

```diff
@@ -109,6 +115,18 @@ class RDKIT_RDGENERAL_EXPORT Dict {
     return *this;
   }
 
+  Dict &operator=(Dict &&other) noexcept {
+    if (this == &other) {
+      return *this;
+    }
+    if (_hasNonPodData) {
+      reset();
+    }
+    _hasNonPodData = std::move(other._hasNonPodData);
```

Comment:
I'm still a bit leery about std::move on POD data types, I assume this is really a copy and nothing is really moved.

Reply:
There's just no need for it. I also forgot to set _hasNonPodData on other to false. Fixing both of those.